docs/source/dev_guide/algorithms/pca/computation-batch.rst

```diff
@@ -0,0 +1,175 @@
+.. ******************************************************************************
+.. * Copyright 2020 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+Batch Processing
+****************
+
+.. contents::
+    :local:
+    :depth: 1
+
+Algorithm Input
+---------------
+The PCA algorithm accepts the input described below. Pass the
+Input ID as a parameter to the methods that provide input for your algorithm.
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+   :align: left
+
+   * - Input ID
+     - Input
+   * - data
+     - Use when the input data is a normalized or non-normalized data set.
+       Pointer to the :math:`n \times p` numeric table that contains the input data set.
+       This input can be an object of any class derived from NumericTable.
+   * - correlation
+     - Use when the input data is a correlation matrix. Pointer to the :math:`p \times p`
+       numeric table that contains the correlation matrix. This input can be an
+       object of any class derived from NumericTable except
```

Comment:
In ALS algorithm the text "This input can be an object of any class ..." is formatted as a Note statement. Should it be formatted the same way across all algorithms?

Reply:
fixed